lib/mission_control/jobs/engine.rb

```diff
@@ -11,7 +11,7 @@ class Engine < ::Rails::Engine
       isolate_namespace MissionControl::Jobs
 
       config.mission_control = ActiveSupport::OrderedOptions.new unless config.try(:mission_control)
-      config.mission_control.jobs = ActiveSupport::OrderedOptions.new
+      config.mission_control.jobs = ActiveSupport::OrderedOptions.new(irb_help: true)
```

Comment:
Why do you need to add this single option here? 🤔

Reply:
That's very odd, you should dig into that instead of just patching it like this, without understanding what's happening.